pyproject.toml

```diff
@@ -126,4 +126,5 @@ typeCheckingMode = "basic"
 reportIncompatibleMethodOverride = false
 reportImportCycles = false
 reportMissingModuleSource = true
+reportMatchNotExhaustive = "error"
```

Comment:
I use of this option to get linter errors to make sure that all code paths with the new roles are covered